addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -473,4 +483,58 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
     .o_survey_choice_question_skipped {
         color: darken($warning, 10%);
     }
+
+    .o_survey_choice_img img {
+        cursor: default;
+        &:hover {
+            box-shadow: none;
+        }
+    }
 }
+
+/**********************************************************
+    Zoomer Specific Style (surveyChoiceImageZoomer widget)
+    When the width is small (mobile), let space above and below
+    to indicate that the user can close it by clicking out.
+ **********************************************************/
+
+.o_survey_img_zoomer_modal {
+    cursor: pointer;
+    .o_survey_img_zoomer_dialog {
+        background-color: rgba(0,0,0,0.65);
+        @include media-breakpoint-down(sm) {
+            height: 80% !important;
+        }
+        .o_survey_img_zoomer_body {
+            img {
+                max-width: 90%;
+                min-width: clamp(250px, 60%, 450px);
+                max-height: 90%;
+                object-fit: contain;
+            }
+            .o_survey_img_zoomer_close_btn {
+                right: 15px;
+                top: 10px;
+                font-size: 1.4rem;
+                z-index: 1;
+            }
+            .o_survey_img_zoomer_btns {
```

Comment:
Would rename all the "zoomer" to simply "zoom".
Also, let's not make this class plural ("btn" and not "btns"), we usually don't do plurals.

Reply:
Ok. Maybe o_survey_img_zoom_tuner, since it contains both buttons?

Reply:
If you use the class on both button tags, then "o_survey_img_zoom_btn" is fine.

If it's used more like a wrapper, then we usually simply go for "o_survey_img_zoom_controls_wrapper" or something like that.

Reply:
In terms of zoom vs zoomer, I've only kept zoomer when it described the component as a whole, let me know what you think :)